### DISCUSSION TITLE
Update examples

### DISCUSSION THREAD

**Discussion by Yunhao0799** [yzhangcs/parser#119] [Q&A]
Hi, 
Would it be possible to update the example.md for semantic dependency parser? I am trying to fine-tune biaffine for Spanish but I think roberta is not available anymore in the latest version

Regards,
Yunhao

**Comment by yzhangcs** [comment]
Hi, Yunhao, apologize for the inconvenience by te missing config. 
But alternatively, you can use `biaffine-dep-roberta-en` instead.
All default args can be freely overrided  from command line. 
Hope this will help you :-).

**Reply by Yunhao0799** [reply]
Thank you for the quick reply :) I'm really new in NLP. But isn't dep for dependency parsing and sdp for semantic dependency parsing?   What problem will raise using one instead of the other?

**Reply by yzhangcs** [reply]
Yeah, they are indeed different tasks, but default args in the config (e.g. lr, batch_size) are the same. You can use them harmlessly.

**Comment by second-brain2030** [comment]
Hi, I'm in education research, is there any chance to fix the probability feature ?
